libraries/ArduinoOTA/ArduinoOTA.cpp

```diff
@@ -170,6 +170,7 @@ String ArduinoOTAClass::readStringUntil(char end){
 void ArduinoOTAClass::_onRx(){
   if(!_udp_ota->next()) return;
   IPAddress ota_ip;
+  IPAddress ota_ip2;
```

Comment:
I kindly ask whether it possible, for readbility, to rename this `to ota_ip_provided_by_cmdline` or `ota_ip_finally_retained` or anything describing what it is and different from "name2".